remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -396,6 +396,49 @@
             "location": "[variables('location')]",
             "tags": {
                 "IotSuiteType": "[parameters('solutionType')]"
+            },
+            "properties": {
```

Comment:
It is a security requirement to not allow any ports open on the VM. We are in process of closing all the ports down by default and add documentation that will allow developers to open ports individually as needed.